egs/swbd/s5c/local/nnet3/tuning/run_tdnn_lfr1b.sh

```diff
@@ -0,0 +1,163 @@
+#!/bin/bash
+
+# _lfr1b is as _lfr1a, but with one more -3,3 layer (the comparable
+# non-LFR system is tdnn_d)
+
+# At this script level we don't support not running on GPU, as it would be painfully slow.
+# If you want to run without GPU you'd have to call train_tdnn.sh with --gpu false,
+# --num-threads 16 and --minibatch-size 128.
+
+# System                   tdnn_d  tdnn_lfr1a  tdnn_lfr1b
+# WER on train_dev(tg)      16.72     17.25     17.00
+# WER on train_dev(fg)      15.31     15.90     15.57
+# WER on eval2000(tg)        19.2      20.1      19.3
+# WER on eval2000(fg)        17.8      18.5      17.8
+# Final train prob       -1.22859  -1.32434  -1.11497
```

Comment:
you can create soft links like run_tdnn_lfr.sh to point to the current best setup.